.classpath

```diff
@@ -24,5 +24,7 @@
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/gcs dependencies"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-appengine-1.18.0-rc.jar"/>
 	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/google-api-client-1.18.0-rc.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-jstl-1.1.2.jar"/>
+	<classpathentry kind="lib" path="src/main/webapp/WEB-INF/lib/repackaged-appengine-jakarta-standard-1.1.2.jar"/>
```

Comment:
It's the version included in the SDK in the dev docs.
I'm not sure up to what version is supported (even natively) yet, I'll probably check that up early next week.